Rebus.AdoNet.Tests/SagaPersisterTests.cs

```diff
@@ -318,31 +399,127 @@ public void WhenIgnoringNullProperties_DoesNotSaveNullPropertiesOnInsert()
 		[Test]
 		public void CanCreateSagaTablesAutomatically()
```

Comment:
Probable the this name is now missleading.. let's find a better one reflecting what this test actually verifies.

Reply:
May SchemaIsCreatedAsExpected